File: programmers/js/level00/120913.js
// 잘라서 배열로 저장하기
function solution(my_str, n) {
  var answer = [];

  for (let i = 0; i < my_str.length; i += n) {
    answer.push(my_str.slice(i, i + n));
  }

  return answer;
}

// 다른 풀이
// function solution(my_str, n) {
//   return my_str.match(new RegExp(`.{1,${n}}`, "g"));
// }

File: programmers/java/level00/120913.java
// 잘라서 배열로 저장하기

class Solution {
    public String[] solution(String my_str, int n) {
        int len = (int) Math.ceil((double) my_str.length() / n);
        String[] result = new String[len];
        
        for (int i = 0; i < len; i++) {
            int end = i * n + n >= my_str.length() ? my_str.length() : i * n + n;
            result[i] = my_str.substring(i * n, end);
        }
        
        return result;
    }
}